src/engraving/rw/read206/read206.cpp

```diff
@@ -3028,9 +3028,9 @@ static void readBox(Box* b, XmlReader& e, ReadContext& ctx)
     b->setBottomMargin(0.0);
     b->setTopGap(Millimetre(0.0));
     b->setBottomGap(Millimetre(0.0));
-    b->setAutoSizeEnabled(false);
-    b->setPropertyFlags(Pid::TOP_GAP, PropertyFlags::UNSTYLED);
-    b->setPropertyFlags(Pid::BOTTOM_GAP, PropertyFlags::UNSTYLED);
+    b->setAutoSizeEnabled(false); // didn't exist in Mu1
+    //b->setPropertyFlags(Pid::TOP_GAP, PropertyFlags::UNSTYLED);
+    //b->setPropertyFlags(Pid::BOTTOM_GAP, PropertyFlags::UNSTYLED);
```

Comment:
What is the reason for these changes?

Reply:
Importing Mu1 and Mu2 scores marked BOTOM_GAP of a vertical frame to not be the default setting even if it was

Reply:
Yes, I should have mentioned that abobe, will do now

Reply:
TOP_GAP wasn't an issue, but commenting it out doesn't seem to harm either

Maybe we'd better remove rather than commenting out